app/assets/javascripts/sitewide/form_validators.js.erb

```diff
@@ -22,13 +22,17 @@ $.fn.onSubmitValidator = function(validatorCb) {
  * @param {boolean} clearErr Set clearErr to true if this is the only
  * error that can happen/show.
  */
-function textValidator(ev, textInput, textLimitMin, textLimitMax, clearErr) {
+function textValidator(ev, textInput, textLimitMin, textLimitMax, clearErr, tinyMCE) {
```

Comment:
Will this work everywhere this code is currently called without the last parameter? Just checking...